docs/src/templates/pages/index.hbs

```diff
@@ -35,7 +35,7 @@ This documentation is evolving over time, so if something is missing that you wo
         	<li class="g-col g-span6--narrow g-span4--mid list--tiled-item list--tiled-item--dark">
```

Comment:
Consistent indentation would be nice but not that much of a biggy

Reply:
Yep - have changed